### ISSUE TITLE
Tokenfield CSS style not working properly

### ISSUE THREAD

**Issue by zedtux** [icicletech/bootstrap-tokenfield-rails#10]
I'm facing an issue with the CSS :

![screenshot from 2015-12-28 15-34-24](https://cloud.githubusercontent.com/assets/478564/12020103/99f6699e-ad78-11e5-84f8-a1263a6d9fba.png)

the text field is not extending when long tokens are present or when a lot of tokens are present while the screenshot of the gem show it in the right way.

**Comment by akashdevaraju** [comment]
Hi @zedtux you may have to decrease the margin between the tokens to accommodate this. The input field width `. ui-autocomplete-input ` is calculated first using js and then the token divs are added inside. Decreasing the padding of the input field `.tokenfield` will also help in rectifying it usually by just 5px. 

Im presently not maintaining this gem. you may contact the present maintainer @SrikalaB for further input. 
Thanks

**Comment by zedtux** [comment]
Hey, thank you @akashdevaraju. To be honest I took the CSS from http://sliptree.github.io/bootstrap-tokenfield/ which work perfectly but it's clear to me that the CSS has to work out-of-the-box otherwise people aren't going to use that gem.

**Comment by akashdevaraju** [comment]
We are also using the CSS provided in the original library repo. So technically that should be the same. @SrikalaB can do a comparison of the gem's css and the one on the site and probably put that if there's a difference. 

Thanks for the input @zedtux, appreciated.

**Comment by SrikalaB** [comment]
@zedtux @akashdevaraju I did a diff on the css files in the gem and in the original library repo.  There are no differences.

**Comment by zedtux** [comment]
Hello @SrikalaB and thank you for the check but then I don't understand why it's not working :neutral_face:

**Comment by khskarl** [comment]
Same thing happened to me and fortunately I managed to solve the issue.

Using the inspector I figured `.tokenfield`'s `height: auto` was overridden by a fixed height from another CSS selector, which in my case was `.form-control`.

The problem was the require order in my _application.css_ file, so I made sure bootstrap-tokenfield was the last one to be required.

Changed  

```
 *= require bootstrap-tokenfield
 *= require_tree .
 *= require_self
```

to

```
 *= require_tree .
 *= require bootstrap-tokenfield
 *= require_self
```

It's not a problem with the gem, but it would be useful to write a warning in the readme.
